src/joinRates/helpers.ts

```diff
@@ -0,0 +1,27 @@
+import { Data } from '../quality/model'
+import { DqJoinStatistics, JoinRatesData, JoinRatesTableData } from './model'
+
+export function getTableNames(tableNames?: Array<string>): { 'Table names': Array<string> } | {} {
+  return tableNames ? { 'Table names': tableNames } : {}
+}
+
+export function parseDqResult(dataItem: JoinRatesData): Data {
```

Comment:
We should not need parsing. The schema from the server should be the same as from the library / json file. Is it not?

Reply:
The server keeps this data as a string

Reply:
It should not be needed. Let's chat about it tomorrow. It would be nice to have join rates merged next week.